depthai_nodes/ml/parsers/keypoints.py

```diff
@@ -70,6 +89,14 @@ def setNumKeypoints(self, n_keypoints):
         """
         self.n_keypoints = n_keypoints
```

Comment:
setScaleFactor should set scale_factor not n_keypoints, need to add setNumberKeypoints()

Reply:
It does. When reviewing some lines that are not affected are hidden :)

Reply:
My bad, the hidden lines matched up  perfectly.